src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="navbar">
+        <ul class="navbar__list">
+          <li>
+            <a
+              href="#"
+              class="navbar__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li>
```